build.gradle.kts

```diff
@@ -52,11 +52,17 @@ tasks.withType<KotlinCompile> {
 
 tasks.withType<Test> {
     useJUnitPlatform()
+    testLogging {
+        showStandardStreams = true
+        showExceptions = true
+        showCauses = true
+        exceptionFormat = org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
+    }
```

Comment:
This enables the logging of test failures in console. Because I am using mac and may have different system settings, the exact test behavior is different for my local dev versus github action CI.